app/src/main/java/com/github/swent/echo/data/supabase/SupabaseDataSource.kt

```diff
@@ -86,5 +87,11 @@ class SupabaseDataSource(private val supabase: SupabaseClient) : RemoteDataSourc
         val userTagsSupabase =
             userProfile.tags.map { tag -> UserTagSupabase(userProfile.userId, tag.tagId) }
         supabase.from("user_tags").upsert(userTagsSupabase)
+
+        val associationSubscriptionSupabase =
+            userProfile.associationsSubscriptions.map { association ->
+                AssociationMemberSupabase(userProfile.userId, association.associationId)
```

Comment:
It could be helpful to explain here that `AssociationMemberSupabase` represents an association subscription and not a membership either with a comment or with an additional class named `AssociationSubscriptionSupabase`

Reply:
you're right, I missed to rename that class; it should indeed be named `AssociationSubscriptionSupabase`...
I've changed that